cgroup_top.py

```diff
@@ -190,10 +223,21 @@ def __init__(self, path, base_path):
         self.base_path = base_path
 
     @property
-    def name(self):
+    def short_path(self):
         return self.path[len(self.base_path):] or '/'
 
     @property
+    def name(self):
+        if self.type == 'docker':
```

Comment:
may be add here ``and HAS_DOCKER`` or something similar. People tend to use ctop from a container with only access to the cgroups but no access to the docker binary or socket.